lib/Db/CardMapper.php

```diff
@@ -141,6 +141,23 @@ public function findAllByStack($stackId, $limit = null, $offset = null) {
 		return $this->findEntities($sql, [$stackId], $limit, $offset);
 	}
 
+	public function findAllWithDue($boardId) {
+		$sql = 'SELECT c.* FROM `*PREFIX*deck_cards` c
+	  	INNER JOIN `*PREFIX*deck_stacks` s ON s.id = c.stack_id
+		INNER JOIN `*PREFIX*deck_boards` b ON b.id = s.board_id
+		  WHERE `s`.`board_id` = ? AND duedate IS NOT NULL AND NOT c.archived AND c.deleted_at = 0 AND s.deleted_at = 0 AND NOT b.archived AND b.deleted_at = 0';
+		return $this->findEntities($sql, [$boardId]);
+	}
+
+	public function findMyAssignedCards($boardId, $username) {
```

Comment:
if it is find**My**AssignedCards it should not take a username but just pick up the current one, otherwise please rename to something like findAssignedCardsForUser